README.md

```diff
@@ -3,53 +3,62 @@
 
 Repeats the provided function (sync and async) n times if an error occurs during its execution.
 
+```
+npm i -S @dialonce/catchie
+```
+
+## Config
 
-### Config
-You can set the ``MAX_REPEAT`` env variable to let ``Catchie`` know how many times you want to repeat the function.
+### MAX_RETRY
```

Comment:
@dial-once/developers I don't know if it will be useful to prefixing env var with module name `CATCHIE_MAX_RETRY`. In my point of view it will avoid env var collision. WDYT ?